src/Sentry.Unity.Android/SentryNativeAndroid.cs

```diff
@@ -44,6 +45,13 @@ public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentry
                     // So we register our backend once more to make sure user-defined data is available in the crash report.
                     SentryNative.ReinstallBackend();
                 }
+                ApplicationAdapter.Instance.Quitting += () =>
+                {
+                    options.DiagnosticLogger?.LogDebug("Closing the sentry-java SDK");
+                    SentryJava.Close();
+                    options.DiagnosticLogger?.LogDebug("Closing the sentry-native SDK");
+                    SentryNative.Close();
```

Comment:
I would expect sentry-android to close sentry-native:

https://github.com/getsentry/sentry-java/blob/530d17dec68a4689d4455941a11f2445d8d10116/sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java#L41-L43

Should we close directly here too? 
What happens if we close it twice?

Reply:
shouldn't be an issue from what I can tell but removing the native close